docs/example-select.js

```diff
@@ -0,0 +1,19 @@
+function switchTab(event) {
+    let tab = event.target
+
+    let lang = tab.dataset.lang
```

Comment:
semicolons?

Reply:
Plenty of style guides actively remove them in situations like these

Reply:
Ah we use them in action.js. I'll be consistent.